js/main.js

```diff
@@ -1,115 +1,30 @@
-(function() {
+(function(){
     "use strict";
 
-    function loadPics() {
-        var api_key, method, user_id, format, url, xmlhttp;
+    (function loadPics(){
```

Comment:
Since we're using jQuery, I'd like to see a `$(document).ready()` here instead of a second IIFE. The reason is that the IIFE fires immediately, whether the DOM is complete or not, potentially leading to unavailable elements. By using `$(document).ready()` we can ensure that any DOM selectors or manipulation that we require will be able to access the elements without an issue.